lib/index.js

```diff
@@ -1,9 +1,10 @@
 const util = require('util')
-const {readdir} = require('fs').promises
+const {stat} = require('fs').promises
 
 async function isNodeGypPackage(path) {
-  const files = await readdir(path)
-  return files.some(f => /.*\.gyp$/.test(f))
+  return await stat(`${path}/binding.gyp`)
+    .then(st => st.isFile())
```

Comment:
bonus that this will now fail if someone happened to have a `binding.gyp` directory for some reason